xtask/src/backend.rs

```diff
@@ -0,0 +1,43 @@
+use crate::{nomt::NomtDB, sov_db::SovDB, timer::Timer};
+
+type Value = Vec<u8>;
+type Key = Vec<u8>;
+
+#[derive(Debug, Clone, clap::ValueEnum)]
+pub enum Backend {
+    SovDB,
+    Nomt,
+}
+
+#[derive(Clone, Debug)]
+#[allow(unused)]
+pub enum Action {
+    /// Batch of writes into the storage,
```

Comment:
I am confused, why is it a batch? It seems to be a singular element. Doesn't seem that harmful either.

Reply:
sorry, that comments shouldn't been there. This PR changed a lot of things and I forgot to remove those comments. thanks!